src/averageAges.js

```diff
@@ -15,11 +15,14 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter(
+      ({ sex, died }) => sex === 'm' && Math.ceil(died / 100) === century)
+    : people.filter(({ sex }) => sex === 'm');
+
+  return men.reduce(
+    (prev, { born, died }) => prev + (died - born), 0)
```

Comment:
I recommend using accumulator, acc or total for a parameter of reduce
[MDN Doc](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce)